### ISSUE TITLE
Add a Code of Conduct

### ISSUE THREAD

**Issue by sdruskat** [hermes-hmc/workflow#6]
To make this project a safe environment, we should establish a Code of Conduct. Note that this also requires a respective committee to make the CoC actionable. Members of the committee should provide their individual contact details in the CoC, in case of raised issues with any members of the committee. Additionally, the committee would ideally be made up diversely to lower the reporting barrier.

**Comment by led02** [comment]
SQLite uses [The Rule](https://sqlite.org/codeofethics.html), ich könnte mich aber auch mit anderen Regeln (z.B. adaptiert von [RSE Society](https://society-rse.org/about/policies/) oder auch Bodhisattvacaryāvatāra) vorstellen.

**Comment by poikilotherm** [comment]
I'm all in for a CoC but wouldn't a committee be overkill given the amount of folks currently working on this project?

**Comment by led02** [comment]
Good point. I also realized that it might be a good idea to have something more tailored to software development than to conferences and meetings.

**Comment by sdruskat** [comment]
My $.02: Use something that's widely adopted in FOSS (Contributor's Covenant or similar). Add >=2 personal email addresses instead of a committee (or mailing list), so that people have someone concrete to go to who they know is not the alleged perpetrator (unless both contacts are, in which case we'll have a different problem altogether).
